hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/store/contracts/precompile/SyntheticTxnFactory.java

```diff
@@ -296,8 +297,9 @@ public TransactionBody.Builder createMint(final MintWrapper mintWrapper) {
         return TransactionBody.newBuilder().setTokenMint(builder);
     }
 
-    public TransactionBody.Builder createFungibleApproval(final ApproveWrapper approveWrapper) {
-        return createNonfungibleApproval(approveWrapper, null, null);
+    public TransactionBody.Builder createFungibleApproval(
+            final ApproveWrapper approveWrapper, @NonNull EntityId ownerId) {
```

Comment:
null annotation for approveWrapper missing

Reply:
Fixed.